examples/tutorials/Druggablity_assessment_with_Fpocket.ipynb

```diff
@@ -0,0 +1,2182 @@
+{
```

Comment:
<p>What are the labels? Explain more</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4069/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>